src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/TrackingOptInModal.java

```diff
@@ -170,10 +170,8 @@ private boolean isSuccessfulResponseByUrlPattern(final NetworkTrafficInterceptor
     }
 
     private static String appendTrackingOptOutParameters(String url, String[] instantGlobals) {
-        UrlBuilder urlBuilder = new UrlBuilder();
-
         for (String instantGlobal : instantGlobals) {
-            url = urlBuilder.globallyEnableGeoInstantGlobalOnPage(url, instantGlobal);
+            url = UrlBuilder.createUrlBuilder().globallyEnableGeoInstantGlobalOnPage(url, instantGlobal);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Introduce a new variable instead of reusing the parameter "url". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1226)